### ISSUE TITLE
Bots open doors blocked by master

### ISSUE THREAD

**Issue by Hedgefog** [yapb/yapb#229]
Bots don't use buttons around the door, they just open this door ignoring the master value.

P.S: Maybe it would be better to find buttons that are connected to the door in a specific range and execute "Use" on the button if the button is close enough.

**Comment by ForeskinJohn** [comment]
Is this mod dead or on hiatus?

**Comment by jeefo** [comment]
Nope, I'm in process of getting mortgage and moving to new location with a family. No free time atm, nor access to normal pc.

As soon as I finish, I wiill return to bots)

**Comment by ForeskinJohn** [comment]
fair enough, i have a child on the way myself and have found a lot less time to do really anything unless im on free or spare time but these bots still kick ass, best for 1.6 imo.
